modules/data/event-hub/main.bicep

```diff
@@ -0,0 +1,245 @@
+@description('Optional. Name for the Event Hub cluster, Alphanumerics and hyphens characters, Start with letter, End with letter or number.')
+param clusterName string = 'null'
```

Comment:
```suggestion
param clusterName string = ''
```

Reply:
I was trying to do the same, But I was getting error BadRequest - Deployment template validation failed: 'The template resource '' of type 'Microsoft.EventHub/clusters' at line '1' and column '17475' is not valid. The name property cannot be null or empty.

So I kept it's value 'null'

Reply:
yikes. Can you poke a comment in with that explination so I dont forget that

Reply:
Sure, Added it